tests/gordo/cli/test_cli.py

```diff
@@ -283,11 +281,11 @@ def test_build_cv_mode(
     ],
 )
 def test_build_cv_mode_cross_val_cache(
+    tmpdir,
```

Comment:
^ :squid:

Reply:
blah, I purposely didn't do that because it's not in `typing` but rather import from pathlib I think, and would require importing that in every file which uses `tmpdir` as a fixture.. maybe we should though. @flikka ?